stellargraph/mapper/mini_batch_node_generators.py

```diff
@@ -144,7 +143,7 @@ def __init__(self, G, clusters=1, q=1, lam=0.1, name=None):
             print(f"{i} cluster has size {len(c)}")
 
         # Get the features for the nodes
-        self.features = G.node_features(self.node_list)
+        self.features = G.node_features(self.node_list[:1])
```

Comment:
This change invalidates the comment, given its now just the node features of a single node and is more than a little confusing. Could you update the comment to discuss more specifically what's going on?

Potentially we could even change this to do what it "really means" rather than store a subset of the feature vectors (since storing a subset is apparently fine)?

Reply:
Oh yep I'll update the comment to what I previously had in Neo4jClusterGCN.

Yeah the proper fix would be to just store the feature shape, but it would also now require code changes to ClusterGCN, FullBatchNodeGenerator, FullBatchLinkGenerator, and all the fullbatch methods and didn't want to include in this PR :( I'll also file an issue and chuck in a fixme